### DISCUSSION TITLE
Let user customize coin

### DISCUSSION THREAD

**Discussion by sadiejay** [nickytonline/connect-four#47] [Ideas]
Hey, I was thinking it'd be cool if users could customize their coin piece by color. 

Another cool customization could be to allow users upload an image/text that could be on their coin as well.

I think the main issue with color customization would be to make sure that the user could tell their coins apart from the opponent.

**Comment by nickytonline** [comment]
@sadiejay, I think this is a great idea. I think we would need to make this part of user settings (another feature to discuss probably) for the application. This would be an awesome feature for a version 2 of the app once we have the basic functionality of the game in place.
